diff -ura --color mopidy_tidal/pyproject.toml mopidy_tidal.new/pyproject.toml
--- mopidy_tidal/pyproject.toml	2026-02-15 10:08:14.549835013 -0800
+++ mopidy_tidal.new/pyproject.toml	2026-02-15 10:09:18.116729647 -0800
@@ -47,7 +47,7 @@
 tidallp = "mopidy_tidallp:Extension"
 
 [build-system]
-requires = ["uv_build>=0.9,<0.10.0"]
+requires = ["uv_build>=0.9,<0.11.0"]
 build-backend = "uv_build"
 
 [tool.uv.build-backend]
